.github/workflows/integration-test-gha.yaml

```
@@ -25,65 +25,11 @@ jobs:
       - uses: actions/setup-dotnet@v4
         with:
           dotnet-version: '9.x'
-      - name: Install Task
-        uses: arduino/setup-task@v2
+      - uses: ./.github/actions/setup-k8s
```

actionlint (1.7.4)

description is required in metadata of "Setup and run lexbox k8s" action at "/home/jailuser/git/.github/actions/setup-k8s/action.yml"

(action)